addons/binding/org.openhab.binding.tivo/src/main/java/org/openhab/binding/tivo/internal/service/TivoConfigData.java

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.tivo.internal.service;
+
+import java.util.SortedSet;
+
+/**
+ * The Class {@link TivoConfigData} stores the dynamic configuration parameters used within the {@link TivoHandler } and
+ * {@link TivoConfigStatusProvider}.
+ */
+public class TivoConfigData {
+
+    private String cfgHost = null;
+    private int cfgTcpPort = -1;
+    private int cfgNumConnRetry = 0;
+    private int cfgPollInterval = 30;
+    private boolean cfgPollChanges = false;
+    private boolean cfgKeepConnOpen = false;
+    private int cfgCmdWait = 0;
+    private SortedSet<Integer> cfgIgnoreChannels = null;
+    private String cfgIdentifier = "";
+    private int cfgMinChannel = 1;
+    private int cfgMaxChannel = 9999;
+    private boolean cfgIgnoreChannelScan = false;
+
+    /**
+     * {@link toString} returns each of the configuration items as a single concatenated string.
+     *
+     * @return string
+     * @see java.lang.Object#toString()
+     */
+    @Override
+    public String toString() {
+        return "TivoConfigData [cfgHost=" + cfgHost + ", cfgTcpPort=" + cfgTcpPort + ", cfgNumConnRetry="
+                + cfgNumConnRetry + ", cfgPollInterval=" + cfgPollInterval + ", cfgPollChanges=" + cfgPollChanges
+                + ", cfgKeepConnOpen=" + cfgKeepConnOpen + ", cfgCmdWait=" + cfgCmdWait + ", cfgIgnoreChannels="
+                + cfgIgnoreChannels + ", cfgIdentifier=" + cfgIdentifier + ", cfgMinChannel=" + cfgMinChannel
+                + ", cfgMaxChannel=" + cfgMaxChannel + ", cfgIgnoreChannelScan=" + cfgIgnoreChannelScan + "]";
+    }
+
+    /**
+     * Gets the cfgIdentifier representing the thing name of the TiVo device.
+     *
+     * @return the cfgIdentifier
+     */
+    public String getCfgIdentifier() {
```

Comment:
We are in the configuration class I think this method & field should be better of to be name identifier instead. This also holds for the other occurrences of cfg